src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {

```

Comment:
The `Storage` class should be designed to follow the encapsulation principle. Consider making the `storage` map private and providing necessary operations through public methods to interact with the storage without exposing its internal representation.